salt/states/zabbix_host.py

```diff
@@ -224,17 +250,53 @@ def _interface_format(interfaces_data):
         for k, v in inv_item.items():
             new_inventory[k] = str(v)
 
+    if "visible_name" not in kwargs:
+        visible_name = None
+    else:
+        visible_name = kwargs.pop("visible_name")
```

Comment:
`visible_name = kwargs.pop("visible_name", None)`

Reply:
Thank you! This change makes the code easier to follow.